pkg/plugin/grpc.go

```diff
@@ -0,0 +1,45 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package plugin
+
+import (
+	"context"
+
+	jvspb "github.com/abcxyz/jvs/apis/v0"
+)
+
+// GRPCClient is an implementation of Validator that talks over RPC.
+type GRPCClient struct{ client jvspb.JVSPluginClient }
+
+func (m *GRPCClient) Validate(justification *jvspb.Justification) (*jvspb.ValidateJustificationResponse, error) {
```

Comment:
I think the signature should be the same as the client.Validate.

Reply:
Updated in https://github.com/abcxyz/jvs/pull/290